apple-codesign/src/cli/certificate_source.rs

```diff
@@ -263,6 +268,79 @@ impl KeySource for MacosKeychainSigningKey {
     }
 }
 
+#[derive(Args, Clone, Debug, Deserialize, Eq, PartialEq)]
+pub struct WindowsStoreSigningKey {
+    /// (Windows only) Windows Store to operate on
+    #[arg(long = "store-name", value_parser = crate::cli::STORE_NAMES, value_name = "STORE")]
```

Comment:
`store` is somewhat ambiguous. Please change these arguments to be prefixed by `windows-store-`.